src/components/ColumnsConfiguration/ColumnsConfiguration.react.js

```diff
@@ -112,13 +112,13 @@ export default class ColumnsConfiguration extends React.Component {
               <div className={styles.footer}>
                 <Button
                   color='white'
-                  value='Hide All'
+                  value='Hide all'
                   width='85px'
                   onClick={this.hideAll.bind(this)} />
                 <Button
                   color='white'
                   value='Show all'
-                  width='85px'
+                  width='90px'
```

Comment:
Could this just use auto width?

Reply:
I removed the width on this buttons and removed the min-width of 110px on button css, in another issue, I will come back to the UI to try to accommodate the buttons better.